src/messages/MOSDOp.h

```diff
@@ -267,7 +261,9 @@ struct ceph_osd_request_head {
   }
 
   virtual void decode_payload() {
-    bufferlist::iterator p = payload.begin();
+
+    std::cout << "Kaczka - wersja: " << header.version << std::endl;
```

Comment:
This looks like a debugging statement that need to be removed now? Or did you want to leave it? In which case it needs to be English and use the dout macro system. :)